docs_outdoornav_user_manual/features/assisted_teleoperation.mdx

```diff
@@ -0,0 +1,101 @@
+toc_max_heading_level: 4
+---
+
+:::danger SAFETY WARNING
+
+Making changes to any of the following variables will decrease system safety. It is recommended that users
+only modify these parameters in consultation with [Clearpath Robotics Support](../support.mdx).
+
+:::
+
+:::note
+
+Changes to any of the following variables will only take effect after power cycling the UGV.
+
+:::
+
+
+As of version 0.12.0, the OutdoorNav software contains an assisted teleoperation feature that will help users
+teleoperate their UGV safely, without worry of colliding into obstacles.
+
+<center>
+  <figure>
+    <img
+      src="/img/outdoornav_images/assisted_teleop_twist_mux.png"
+      width="1000"
+    />
+    <figcaption>Block Diagram of Assisted Teleoperation Twist Mux</figcaption>
+  </figure>
+</center>
+
+
+The assisted teleoperation features is intended to work with on the following sources of velocity input:
+
+| Input source | Topic  |
+|--------------|--------|
+| UI Joystick         | `/ui_teleop/cmd_vel` |
+| PS4 Controlller     | `/bluetooth_teleop/cmd_vel` or `/joy_teleop/cmd_vel`|
+| Logitech Controller | `/joy_teleop/cmd_vel` | 
+
+
+The Assisted teleop feature will **NOT** work on the following sources of velocity input:
+
+| Input source | Topic |
+|--------------|-------|
+| External Command   | `/cmd_vel` |
+| Keyboard Command   | `/kb_teleop/cmd_vel` |
+| Interactive Marker | `/twist_marker_server/cmd_vel` |
+
+The Assisted Teleoperation feature can be activated/disactivated according to your required application requirements.
+This can be done by setting the **ENABLE_ASSISTED_TELEOP** environment variable in the 
+`/opt/onav/<onav_version>/config/autonomy.env` file, where *onav_version*
+is the currently running version of OutdoorNav.
+
+## Functionality
+
+The intended use for 
+
+For visualization details of the assisted teleoperation feature, please consult the [UI Manual Mode](../web_user_interface/ui_manual_mode.mdx) page.
+
+
+### Enable/Disable from the UI
+
+From the OutdoorNav UI you are able to enable/disable the "Teleop Assist" by using the toggle that is located by the UI joystick.
+
+### Enable/Disable using Joystick
+
+To disable the "Teleop Assist" when driving the UGV with the PS4 or Logitech controllers, you will need to press and hold the following button:
+
+| Controller Type  | Button |
+|--------------|-------|
+| PS4 controller   | **O** |
+| Logitech controller   | **B** |
+
+### Error/Warnings
+
+OutdoorNav will throw warnings and display them on the UI related to the status of the assisted teleoperation feature.
+Below are the errors/warnings you may see:
+
+| Error/Warning | Reason |
+|---------------|--------|
+| **"Error: Failed to communicate with the Assisted Teleop node"** | The assisted teleoperation node is not running or has crashed |
+| **"Error: No sensors configured"**  | The assisted teleoperation node is running but no sensors have been configured to be used as obstacle  detection sources |

```

Comment:
Super minor but it looks like there are 2 spaces between obstacle and detection.